src/main/java/omero/gateway/Connector.java

```diff
@@ -681,8 +681,9 @@ void joinSession()
      * 
      * @param networkup Pass <code>true</code> if the network is up,
      * <code>false</code> otherwise.
+     * @param closeSession Pass <code>true</code> to close the session
      */
-     void close(boolean networkup)
```

Comment:
For this not to be a breaking change, you'll need to keep the old method signature and call the new method with a default (optionally adding a `Deprecated` annotation)

Reply:
Can do. But it's not a public method, the Connector class isn't really meant to be used directly, only to be used by the Gateway.

Reply:
Hmm... ok. Fair enough.

Reply:
But I'll add the previous method as deprecated again, doesn't do any harm.